src/System.Text.Utf8/src/System/Text/Utf16/Utf16LittleEndianEncoder.cs

```diff
@@ -7,7 +7,7 @@ public static class Utf16LittleEndianEncoder
     {
         const uint MaskLow10Bits = 0x3FF;
 
-        public static bool TryDecodeCodePoint(ByteSpan buffer, out UnicodeCodePoint codePoint, out int encodedBytes)
+        public static bool TryDecodeCodePoint(Span<byte> buffer, out UnicodeCodePoint codePoint, out int encodedBytes)
```

Comment:
ByteSpan was Krzysztof's C. first implementation which is a simply byte* and length.
Slices/Span<byte> is someone else implementation of the same thing. We are testing which version is faster and I believe we will remove one of them.